notebooks/official/custom/get_started_vertex_training_xgboost.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<p>I've noticed that more notebooks are using <code>%</code> instead of <code>!</code> for invoking pip.</p><p> </p><p>Nit: I would also suggest keeping with either single character flags or complete  argument names (e.g. <code>--upgrade --quiet</code> or <code>-qu</code> .  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Yes, convention in notebooks is to use ! to execute in the outer linux shell. % is a builtin.</p>